kubernetes/INSTALL.md

```diff
@@ -7,11 +7,11 @@
     
     kubectl config set-context `kubectl config view | grep current-context | awk '{print $2}'` --namespace <ns>
 
-**Install Istio infra**
+**Install Istio control plane and management plane**
 
```

Comment:
In our case, we do not have a management plane yet. Manager and mixer are still control plane elements, configuring the network of proxies or controlling traffic. A true management plane is the API layer and associated set of UI services, toolchain services (ci/cd), etc. that is the plane the user interacts with, provides the configs etc. and that's the plane that kicks off staged rollouts (of versions or configs), does sanity checks, etc etc. 

If you are referring to the networking model of setting up snmp, ssh etc to configure the routers, then that totally doesn't apply to us. Networking management plane is out of band access to the devices in data plane, does not have to touch SDN control plane to manipulate forwarding rules. Istio management plane should always go through the control plane to ensure consistent, validated and safe access to the istio deployment as a whole. Out of band access would negate all safeguards we have. 

Our control plane, like SDN control plane consists of an entity that sets up forwarding tables proactively. It also does policy enforcement on the data path (still control plane).

Our management plane would be the tools we deploy on top of istio control plane. 
My two cents. 

And istio infra sounds too colloquial. Infrastructure doesn't seem to fit well in this context given that the term is already used to typically mean the underlying cloud infrastructure.